Assets/MixedRealityToolkit/_Core/Definitions/InputTypeDatum/SixDoF.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem;
+using System;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.InputTypes
+{
+    public struct SixDoF : IMixedRealityInputTypeDatum<Tuple<Vector3, Quaternion>>
```

Comment:
SixDegreesOfFreedomInputType?  Abbreviations in type name aren't much cared for in .net